docs/content/latest/troubleshoot/cluster/failed_tablets.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Recover failed YB-TServer for failed tablets
```

Comment:
I would probably call this `Recover Tserver from crash loop` to be more precise, as we now automatically tolerate tablets independently failing and getting remote bootstrapped, while this page will be specifically about crashes